core/templates/dev/head/pages/Base.ts

```diff
@@ -20,7 +20,6 @@ require('services/contextual/DocumentAttributeCustomizationService.ts');
 require('services/contextual/MetaTagCustomizationService.ts');
 require('services/contextual/UrlService.ts');
 require('services/stateful/BackgroundMaskService.ts');
-
```

Comment:
Don't remove this line.

Reply:
I am not sure but I think you are mentioning about a new line which i removed and so I added. If I am wrong please correct me.